src/main/java/hudson/plugins/templateproject/ProxyBuilder.java

```diff
@@ -45,16 +34,22 @@ public String getProjectName() {
 		return projectName;
 	}
 
-	public String getExpandedProjectName(AbstractBuild<?, ?> build) {
```

Comment:
It is possible, but unfortunately it does not help much. The API method may be used in closed-source plugins and in System Groovy Scripts in Jenkins, so a plugin check is not enough to guarantee the compatibility.

If you feel the method should not be public, do the following:
* Mark the method as <code>@Deprecated</code>
* Add the <code>@Restricted(NoExternalUse.class)</code> annotation. It will prevent dependent plugins from compiling with the new plugin versions if they use this method